sdk/python/foundation-models/system/inference/text-generation/text-generation-online-endpoint-v2.ipynb

```diff
@@ -238,12 +242,13 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "# score the sample_score_dolly.json file using the online endpoint with the azureml endpoint invoke method\n",
+    "# score the payload file using the online endpoint with the azureml endpoint invoke method\n",
     "response = workspace_ml_client.online_endpoints.invoke(\n",
     "    endpoint_name=online_endpoint_name,\n",
     "    deployment_name=\"demo\",\n",
```

Comment:
rename deployment demo to textgenerationdp or some better name in all instances.

Reply:
@skasturi @novaturient95 Correct, these are not run in GitHub actions currently.
If you run the file sdk\python\readme.py it generates GitHub action files for these notebooks but they have not been checked in.

Reply:
> rename deployment name from demo to (textgenerationdp or some better name) in all instances.

Done